include/zephyr/canbus/isotp.h

```diff
@@ -275,7 +275,7 @@ void isotp_unbind(struct isotp_recv_ctx *rctx);
  * @param len     Size of the buffer.
  * @param timeout Timeout for incoming data.
  *
- * @retval Number of bytes copied on success
+ * @return Number of bytes copied on success
  * @retval ISOTP_RECV_TIMEOUT when "timeout" timed out
  * @retval ISOTP_N_* on error
```

Comment:
It's a bit curious to use both `@return` and `@retval`, but I guess it works: 

<img width="677" height="229" alt="Image" src="https://github.com/user-attachments/assets/04fe5cdf-076a-4e7f-a7dc-1c12daed7ba3" />

Although it may be better to have something like 

<img width="841" height="623" alt="Image" src="https://github.com/user-attachments/assets/99a19649-2b20-4a05-8329-f5e53a8ccca6" />
(from e.g. https://man7.org/linux/man-pages/man2/send.2.html) 

i.e. making it clear that it either returns a value, or a set of errors. I don't see anything in doxygen that supports this though - @kartben thoughts?

Reply:
> I think "Returns" vs. "Returns values" is already not so bad though?

I don't think it is bad, I just think "Return Value" and "Errors" is better :)